DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/EditDelegateGroupDescription.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateGroups
+@using Microsoft.Extensions.Configuration
+@model EditDelegateGroupDescriptionViewModel
+
+@{
+  ViewData["Title"] = $"Edit description for {Model.GroupLabel} group (optional)";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/delegates/groupDelegates.css")" asp-append-version="true">
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
```

Comment:
Extra thing I didn't know about until the review on one of my tickets, can you use the _AutoNavMenuItems partial instead.
`<partial name="_AutoNavMenuItems" model="ApplicationType.TrackingSystem"/>`

Reply:
Sorry, I didn't know this. I updated the code to as above partial view